untrack/src/lib.rs

```diff
@@ -76,21 +72,51 @@ impl Args for Options {
 }
 
 pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
-    let (urn, _) =
+    let (urn, repo) =
         project::cwd().context("this command must be run in the context of a project")?;
     let profile = ctx.profile()?;
 
-    execute(&urn, options, &profile)
+    execute(&urn, Some(&repo), options, &profile)
 }
 
-pub fn execute(urn: &Urn, options: Options, profile: &Profile) -> anyhow::Result<()> {
-    // TODO: Remove remote
-    // TODO: Remove tracking branch
+fn get_peer_id(
+    project: &project::Metadata,
+    storage: &Storage,
+    name: &String,
+) -> anyhow::Result<Option<PeerId>> {
+    let mut found = None;
+    for (id, peer) in project::tracked(project, storage)? {
+        if peer.name() == *name {
+            found = Some(id);
```

Comment:
Don't we want to return here? Ofc. there's also the issue of multiple users with the same name, but can tackle that later..